apis/pkg/v1alpha1/controller_types.go

```diff
@@ -0,0 +1,239 @@
+// Copyright 2024 Upbound Inc.
```

Comment:
```suggestion
// Copyright 2025 Upbound Inc
```

We may do the same elsewhere.